index.js

```diff
@@ -1,21 +1,19 @@
 const { writeFileSync } = require('fs');
 
 const sass = require('node-sass');
-const inliner = require('sass-inline-svg');
 
-const { IN_FILE, OUT_FILE, SVG_PATH } = require('./CONSTANTS');
+const { IN_FILE, OUT_FILE } = require('./CONSTANTS');
+const functions = require('./funcs');
```

Comment:
May as well recycle this code since we were using it for the CLI use case anyways.